src/Framework/Assert.php

```diff
@@ -2524,6 +2525,20 @@ public static function assertJsonFileNotEqualsJsonFile(string $expectedFile, str
         static::assertThat($actualJson, new LogicalNot($constraintExpected), $message);
     }
 
+    /**
+     * Asserts that given value is resource of specified type
+     *
+     * @param string $expectedType
+     * @param resource $actual
+     * @param string $message
+     *
+     * @throws \Exception
+     */
+    public static function assertResourceOfType(string $expectedType, $actual, string $message = '')
```

Comment:
`assertResourceNotOfType()` is missing

Reply:
maybe there shall be some kind of ~meta test that check if all assertions have theirs negations ?

Reply:
wasn't aware of that, added.